src/components/DynamicInputs.tsx

```diff
@@ -1,13 +1,35 @@
 import React, { useCallback } from 'react';
 
+export enum INPUT_LABEL_TYPE {
+  ADDRESS = 'Address',
+  SCOPE = 'Custom',
+}
+
+const LABEL_PLACEHOLDER = {
+  [INPUT_LABEL_TYPE.ADDRESS]: '0x483b...5f97',
+  [INPUT_LABEL_TYPE.SCOPE]: 'eip155:1',
+};
+
+const INPUT_ID = {
+  [INPUT_LABEL_TYPE.ADDRESS]: 'custom-address-input',
+  [INPUT_LABEL_TYPE.SCOPE]: 'custom-scope-input',
+};
+
+const ADD_BUTTON_ID = {
+  [INPUT_LABEL_TYPE.ADDRESS]: 'add-custom-address-button',
+  [INPUT_LABEL_TYPE.SCOPE]: 'add-custom-scope-button',
+};
```

Comment:
are these constants needed? Or can they be derived from `label` below? i.e. `add-custom-{label}-button`

Perhaps in that case, `label` should be `type` or something else instead

Reply:
yeah that definitely makes sense, since `{label}` has the uppercase char in first index I can just lowercase it and we good 👍🏾

Reply:
resolved in [c2c4d95](https://github.com/MetaMask/test-dapp-multichain/pull/36/commits/c2c4d953cc58698100ed16fe9b2fff2dd83f5c82)